decentraland/unity-renderer
# feat: choose io source device in settings

## What does this PR change?
Closes #2907 

Add the possibility to choose input audio devices through the settings in VoiceChat UI menu

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/choose-io-source-device-in-settings&kernel-branch=feat%2Fchange-io-audio-device
2. Open Setting UI and go to Audio->VoiceChat
3. Select the dropdown to the Input Device. 
4. Go to the web tab and allow your browser to access your devices.
5. Come back to Unity and select desired device for voice chat
6. Check that the selected device is used for the voice chat

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
